README.md

```diff
@@ -4,9 +4,98 @@
 * node - ^12.18.3
 * yarn - ^1.21.1
 
+# Taks 1
+
 * `yarn start:1.1` - task 1.1
 * `yarn start:1.2:ram` - task 1.2 with read file to RAM
 * `yarn start:1.2:stream` - task 1.2 with read file using streams
 
 ## Additional Notes
-`stream.ts` has two variations of implemetation. One with `scvParser.subscribe` and another one with custom transformation. For now it is implemented with just commenting of the second option, but in future I want to deal with conditional streams.
\ No newline at end of file
+`stream.ts` has two variations of implemetation. One with `scvParser.subscribe` and another one with custom transformation. For now it is implemented with just commenting of the second option, but in future I want to deal with conditional streams.
+
+# Task 2
+The goal of the task is to implement service to get/create/update/delete users.
+To run the app use the following command
+* `yarn start:2`
+
+Implemented items
```

Comment:
great that you have such a detailed readme! :muscle: 

Another way would be to create a Swagger doc which would be parsed by Github and look like a real api doc.

Reply:
Yep, I know. But that idea came to me to late 😂